components/script/dom/window.rs

```diff
@@ -311,6 +308,13 @@ pub(crate) struct Window {
     #[cfg(feature = "webxr")]
     webxr_registry: Option<webxr_api::Registry>,
 
+    /// When an element triggers an image load or starts watching an image load from the
+    /// `ImageCache` it adds an entry to this list. When those loads are triggered from
+    /// layout, they also add an etry to [`Self::pending_layout_images`].
+    #[no_trace]
+    #[ignore_malloc_size_of = "A boxed Fn is very hard to measure"]
+    pending_image_callbacks: DomRefCell<HashMap<PendingImageId, Vec<PendingImageCallback>>>,
```

Comment:
Can we pushing the ignore_malloc_size_of onto the actual type that doesn't implement it somehow? It would be useful to see if this hashmap ends up growing unexpectedly, but we can't observe that right now.

Reply:
Yep, this is done now.